app/views/index.erb

```diff
@@ -0,0 +1,21 @@
+<section id="peeps-holder">
+  <ul>
+    <% @peeps.each do |peep| %>
+      <li><%= peep.content %> : <%= peep.author %> on <%= peep.display_date_and_time %></li>
+    <% end %>
+  </ul>
+ </section>
+
+<% if current_user %>
+  <section id="new-peep">
+   <form action="/peeps" method="post">
+    <fieldset>
+      <legend>Please submit a new peep</legend>
+      <p><label>Message: </label><textarea rows="3" name="content"></textarea></p>
```

Comment:
Don't forget that a label needs a for tag if it doesn't wrap around an input or screen-readers won't know which input the label applies to